bindings_ffi/src/mls.rs

```diff
@@ -167,7 +167,7 @@ pub async fn get_inbox_id_for_address(
 #[allow(unused)]
 #[uniffi::export]
 pub fn generate_inbox_id(account_address: String, nonce: u64) -> String {
-    xmtp_id_generate_inbox_id(&account_address, &nonce)
+    xmtp_id_generate_inbox_id(&account_address, &nonce)?
```

Comment:
To use the ? this function needs to return a Result

Reply:
Also works with Optionals